src/components/thumbnail-card/ThumbnailCard.scss

```diff
@@ -13,6 +13,21 @@
     box-shadow: 0 5px 10px 0 rgba(198, 198, 198, .2);
 }
 
+// We should expand the card only when it is hovered or the action item is active
+.thumbnail-card.is-expandable {
```

Comment:
Should this be namespaced and SUIT styled?

Reply:
I can make it say `is-expanded` instead since I think that's how the SUIT convention works